docker-compose.yml

```diff
@@ -32,6 +32,56 @@ services:
   rosie:
     image: serenata/rosie
 
+  prometheus:
+    image: arthursens/serenata-prometheus
+    ports: 
+    - 9090:9090
+    links:
+    - node_exporter:node_exporter
+    - process_exporter:process_exporter
+    - postgres_exporter:postgres_exporter
+    - bi_exporter:bi_exporter
+
+  grafana:
+    image: arthursens/serenata-grafana
+    links:
+    - prometheus:prometheus
+    - postgres:postgres
+    ports:
+    - 3000:3000
+
+  node_exporter:
+    image: prom/node-exporter
+
+  process_exporter:
+    image: ncabatoff/process-exporter
+    privileged: yes
+    volumes: 
+    - /proc:/host/proc
+    - ./prometheus/process-exporter:/config
+    command: --procfs /host/proc -config.path /config/config.yml
+
+  postgres_exporter:
+    image: wrouesnel/postgres_exporter
+    environment:
+      DATA_SOURCE_NAME: postgres://jarbas:mysecretpassword@postgres/jarbas?sslmode=disable
+    links:
+    - postgres:postgres
+    volumes:
+    - ./prometheus/postgres-exporter:/prometheus
+    command: --extend.query-path /prometheus/monitoring.yml
+
+  bi_exporter:
+    image: wrouesnel/postgres_exporter
+    environment:
+      DATA_SOURCE_NAME: postgres://jarbas:mysecretpassword@postgres/jarbas?sslmode=disable
+    links:
+    - postgres:postgres
+    volumes:
+    - ./prometheus/postgres-exporter:/prometheus
+    command: --disable-default-metrics --disable-settings-metrics --extend.query-path /prometheus/businessInteligence.yml
```

Comment:
This agent uses the same image as [postgres_exporter](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-4e5e90c6228fd48698d074241c2ba760R65), but it has different configurations.

First I've disabled the monitoring queries with the `--disable-default-metrics --disable-settings-metrics` flags
The YAML file with the custom flags is also different, as you can see [here](https://github.com/okfn-brasil/serenata-de-amor/pull/484/files#diff-1f8c3941afc2afb592b6ca21368a4961R1)